src/js/themes/hpe.js

```diff
@@ -251,6 +251,33 @@ export const hpe = deepFreeze({
         },
         color: undefined,
       },
+      extend: ({ colorValue, primary, theme }) => `
```

Comment:
Can we put extend at the end of this block just to minimize emphasis on extend?

Reply:
yup!